scripts/monitoring/cron-send-user-count.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+COUNT=$(oca get --no-headers users | wc -l)
+echo
+echo "Number of users : $COUNT"
```

Comment:
Can we get output from this script only if --verbose/-v is passed? That way we do not get spammed from cron's output.